prov/tcp/src/tcpx_comm.c

```diff
@@ -57,42 +57,29 @@ int tcpx_send_msg(struct tcpx_pe_entry *pe_entry)
 	return FI_SUCCESS;
 }
 
-static int tcpx_recv_msg_hdr(struct tcpx_pe_entry *pe_entry)
+int tcpx_recv_field(SOCKET sock, void *buf, size_t buf_len,
+		    uint64_t *done_len, size_t start_offset)
```

Comment:
It looks odd seeing both done_len and start_offset.  Why doesn't done_len == start_offset?